packages/core/src/actions/stopChild.ts

```diff
@@ -92,7 +92,7 @@ export interface StopAction<
  *
  * @param actorRef The actor to stop.
  */
-export function stop<
+export function stopChild<
```

Comment:
we might want to export a deprecated `stop` alias from the `src/index.ts` for this